docs/userGuide/contentAuthoring.md

```diff
@@ -576,4 +576,12 @@ Note:
 - Footers should not be nested in other components or HTML tags. If found inside, they will be shifted outside to be rendered properly.
 - [MarkBind components](#use-components) and [includes](#include-contents) are not supported in footers.
 
-</div>
+### Using Anchors

```

Comment:
The word "Using" seems a bit weird because the author doesn't have to use it, it is automatically provided for them.

Also, this is a feature showcase, but the rest of the page is about how to utilize MarkBind features. But I couldn't find a suitable place in the documentation to move this section to, so I think it's fine... 🤔